### ISSUE TITLE
Can't restore with MFA authentication enable (customer side)

### ISSUE THREAD

**Issue by SebastienMenard** [VeeamHub/vbo365-rest-self-service#6]
Hi Niels,

A big thank you for your development work around the auto restoration portal!

A small problem unfortunately:
When I want to use modern authentication to connect my organization to the VBO, the self-healing portal does not work.

Let me explain:
Vbo server side, everything is fine, backup and restore.
The self-healing portal works well with my administrator credentials and my organization (when I use Basic authentication)

However, when I use the organization credentials (with mfa access) to log in to the self-healing portal, I can log in, but when I want to use a restore point, I can not select a restoration for the year 1970.

I can not start a restore session, I have the following message:
Error starting restore session
Error: An unsecured or poorly secured error has been received from the other party. See the internal FaultException for the error code and details.

Do you have an idea of ​​what's wrong?

Thank you

**Comment by nielsengelen** [comment]
It should work for MFA as well as that is what I use for testing. Can you check the RESTful API logs for any errors - more specific for: "Failed to load plug-in"? If this is the case, can you do a re-install of the explorers and restart the VBO services (all 3 of them)?

You can find them in the “C:\ProgramData\Veeam\Backup365\Logs” folder and are named like:
Veeam.Archiver.REST_2019_11_15_10_43_04.log

**Comment by SebastienMenard** [comment]
Hello,

I found the mistake. I moved my organization from basic authentication to modern authentication.
When I have wanted to log in to the self-service portal, I used the user login/ user password (such as Basic authentication) instead of the user / application password !

Maybe because I have move from the basic application to the modern application, the self restore portal the portal let me authenticate (with the user's password) before restarting vbo services but not after (authentication failure).

So, thank you to your help !
